### PR TITLE
Create initial analytics function

### PR DESCRIPTION
**Pull request by EstebanDalelR**
## Description
Initial analytics to know when the user does stuff (and if it breaks!)

## Type of change
- New feature (non-breaking change which adds functionality)

### CODE REVIEW

**Comment by EstebanDalelR** [comment]
Should we save owner and repo?

File: src/utils/analytics/searchType.ts [hunk 1]
```diff
@@ -0,0 +1,21 @@
+const axios = require("axios");
+
+export default async function searchType({
+  searchType,
+  owner,
+  repo,
+}: {
+  searchType: string;
+  owner?: string;
```
Comment (EstebanDalelR) [inline src/utils/analytics/searchType.ts:9]: should this come?

**Review by baristaGeek — COMMENTED** [review]
I don't know how to test this. I pulled the changes on the on the other repo's branch, ran locally on port 3001 and ran a query on our extension using this branch. I don't see anything change on the Analytics base on Airtable. 

Please elaborate on this.

**Comment by EstebanDalelR** [comment]
With the backend running and connected to airtable in the correct branch, do a `watermelon.start` query.  
It should request to your localhost in the port 3001 and that should send it to the db.

**Review by baristaGeek — APPROVED** [review]

**Pull request merged**